### PR TITLE
refactor(core): convert create pipeline to react

### PR DESCRIPTION
**Pull request by jrsquared**

### CODE REVIEW

File: app/scripts/modules/core/src/delivery/create/CreatePipelineButton.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      showCreatePipelineModal: false,
+    };
+  }
+
+  private showCallBack(showCreatePipelineModal: boolean) {
+    this.setState({showCreatePipelineModal});
+  }
+
+  private createPipeline() {
+    ReactGA.event({category: 'Pipelines', action: `Create Pipeline`});
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/src/delivery/create/CreatePipelineButton.tsx]: why backticks?

Comment (jrsquared) [reply app/scripts/modules/core/src/delivery/create/CreatePipelineButton.tsx]: The place I copy pasted it from had a variable and I didn't change to quotes.

File: app/scripts/modules/core/src/pipeline/config/CreatePipeline.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+import { CreatePipelineButton } from 'core/delivery/create/CreatePipelineButton';
+
+export interface ICreatePipelineProps {
+  application: Application;
+}
+
+@BindAll()
+export class CreatePipeline extends React.Component<ICreatePipelineProps> {
+  private dropdownToggled(): void {
+    ReactGA.event({category: 'Pipelines', action: `Configure (top level)`});
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/src/pipeline/config/CreatePipeline.tsx]: backticks?

File: app/scripts/modules/core/src/pipeline/config/CreatePipeline.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+          {application.strategyConfigs.data.map((pipeline: any) => <Pipeline key={pipeline.id} pipeline={pipeline} type="strategy"/>)}
+          </Dropdown.Menu>
+      </Dropdown>
+    );
+  }
+}
+
+const Pipeline = (props: { pipeline: any, type: 'pipeline' | 'strategy' }): JSX.Element => {
+  const clicked = () => {
+    ReactGA.event({category: 'Pipelines', action: `Configure ${props.type} (via top level)`});
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/src/pipeline/config/CreatePipeline.tsx]: nice use of backticks

Comment (jrsquared) [reply app/scripts/modules/core/src/pipeline/config/CreatePipeline.tsx]: third times the charm

Comment (anotherchrisberry) [reply app/scripts/modules/core/src/pipeline/config/CreatePipeline.tsx]: 🥉

**Pull request merged**